client/blocks/comments/comment-actions.jsx

```diff
@@ -21,7 +21,7 @@ const CommentActions = ( {
 	handleReply,
 	onReplyCancel,
 	showReadMore,
-	onReadMore,
+	onReadMore = noop,
```

Comment:
Unnecessary - `onClick` is optional and it's fine if it's `undefined`. Not to mention that `showReadMore` prop is unnecessary here and it should rely on the presence of `onReadMore`...